### PR TITLE
+ extra info about reason for DDL, etc.

### PR DESCRIPTION
**Pull request by Dhghomon**

### CODE REVIEW

**Review by 1st1 — APPROVED** [review]

**Comment by 1st1** [comment]
@Dhghomon Dave, can you please accept the CLA when you have a chance?

File: docs/tutorial/queries.rst [hunk 1]
```diff
@@ -98,7 +98,9 @@ the query shape. The year for the movie "Dune" is given as ``{}`` (the
 empty set) since no ``year`` is set for that object.
 
 Let's narrow down the ``Movie`` search to "blade runner" using
-:eql:op:`ILIKE` (simple case-insensitive pattern matching):
+:eql:op:`ILIKE` (simple case-insensitive pattern matching). With the %
+at the end anything after `blade runner` will match (Blade Runner,
```
Comment (1st1) [inline docs/tutorial/queries.rst:102]:
```suggestion
at the end anything after ``blade runner`` will match (Blade Runner, 
```

**Review by 1st1 — APPROVED** [review]

**Comment by Dhghomon** [comment]
> @Dhghomon Dave, can you please accept the CLA when you have a chance?

Worked for this one! The CLA elsewhere is giving a server error sometimes but I'll keep trying.
image

**Comment by Dhghomon** [comment]
> > @Dhghomon Dave, can you please accept the CLA when you have a chance?
> 
> Worked for this one! The CLA elsewhere is giving a server error sometimes but I'll keep trying.
> image

Going to have to try again a bit later with the signing - still getting server errors every time.

**Comment by 1st1** [comment]
> Worked for this one! The CLA elsewhere is giving a server error sometimes but I'll keep trying.

@elprans ^

**Comment by elprans** [comment]
The 500 errors are due to broken database connections.  We need to release edgedb-js with RFC1004 and port the CLA bot to use it.

**Pull request merged**